### DISCUSSION TITLE
Bag broken

### DISCUSSION THREAD

**Discussion by Fordowner49** [nroutasuo/level13#297] [Q&A] [answered]
Where can I find a new bag or elastic bands to fix mine?

**Comment by Fordowner49** [comment]
I found a new bag. I am still wanting to know where I can find elastic bands though.

**Reply by nroutasuo** [reply, marked as answer]
It might take awhile, it depends on your map, but eventually you will find them. Look for spots with this icon on the map. (![map-ingredient](https://github.com/nroutasuo/level13/assets/1489503/d8707ea1-ce3d-4051-ab6b-0e2fd6b94847))

**Reply by Fordowner49** [reply]
Thank you! since asking the question i have unlocked leather working and have a backpack now
